src/ploneintranet/docconv/client/indexers.py

```diff
@@ -0,0 +1,11 @@
+from Products.CMFCore.interfaces import IContentish
+from plone.indexer import indexer
+from ploneintranet.docconv.client.interfaces import IDocconv
+
+
+@indexer(IContentish)
+def has_thumbs(obj):
+    """
+    Does this item have preview thumbnails?
```

Comment:
I always find questions as docstrings a little odd. It would be better to follow the PEP0257 convention:
```
The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
```